eng/templates/stages/release.yml

```diff
@@ -78,7 +78,8 @@ stages:
 
   - ${{ if eq(parameters.dotnetMajorVersion, '8.0') }}:
     - name: sourceBuildArtifactLeg
-      value: CentOSStream8_Offline_x64_Artifacts
+      # When .NET 8.0 is released, gather artifacts from the `PreviousSourceBuiltSdk` leg instead
```

Comment:
Maybe make this a TODO and link to an issue?

Reply:
I personally think an issue should be logged and the comment removed.

Reply:
Filed https://github.com/dotnet/source-build/issues/3304